### PR TITLE
apply editor feedback

### LINKED ISSUE #1083: Audit #15 of 20 top viewed articles - "Outlook Web Access (OWA) for Exchange"
Beau's team is performing the SME check on all of the Office articles. 
https://support.rackspace.com/how-to/outlook-web-access-owa-for-exchange/

- [x]  SME check
- [x]  Editor check
- [x]  Update the 'audit_date' and 'last_modified_by' metadata
- [x]  If any changes were made, update the 'last_modified_date' metadata

### PR DESCRIPTION
**Pull request by stephamon**
Fixes #1083

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e3bf2457d5/how-to/)

File: content/exchange/outlook-web-access-owa-for-exchange.md [hunk 1]
```diff
@@ -1,257 +1,171 @@
...
+options.
 
--   Shortcuts - This section allows you to set up
-    your Out of Office notification and also access the auto
-    configuration tool to set up your Outlook
-    email client.
+**Account**
 
-<img src="{% asset_path exchange/outlook-web-access-owa-for-exchange/OWA2.png %}" alt="" />
+-   Account Information: Update your personal information such as your display name, contact information, and and access settings for email clients to set up Exchange with IMAP and POP access.
```
Comment (kylelaffoon) [inline content/exchange/outlook-web-access-owa-for-exchange.md]: @stephamon We have an extra "and" in here.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-eef744f331/how-to/). :bow:

**Comment by kylelaffoon** [comment]
LGTM

**Pull request merged**